tools/specs/Microsoft.Data.SqlClient.nuspec

```diff
@@ -110,27 +110,27 @@ When using NuGet 3.x this package requires at least version 3.4.</description>
     <references>
       <group targetFramework="net462">
           <reference file="Microsoft.Data.SqlClient.dll" />
-          <reference file="Microsoft.Data.SqlClient.pdb" />
+          <!-- <reference file="Microsoft.Data.SqlClient.pdb" /> -->
           <reference file="Microsoft.Data.SqlClient.xml" />
       </group>
       <group targetFramework="net7.0">
           <reference file="Microsoft.Data.SqlClient.dll" />
-          <reference file="Microsoft.Data.SqlClient.pdb" />
+          <!-- <reference file="Microsoft.Data.SqlClient.pdb" /> -->
           <reference file="Microsoft.Data.SqlClient.xml" />
       </group>
       <group targetFramework="net6.0">
           <reference file="Microsoft.Data.SqlClient.dll" />
-          <reference file="Microsoft.Data.SqlClient.pdb" />
+          <!-- <reference file="Microsoft.Data.SqlClient.pdb" /> -->
           <reference file="Microsoft.Data.SqlClient.xml" />
       </group>
       <group targetFramework="netstandard2.0">
           <reference file="Microsoft.Data.SqlClient.dll" />
-          <reference file="Microsoft.Data.SqlClient.pdb" />
+          <!-- <reference file="Microsoft.Data.SqlClient.pdb" /> -->
           <reference file="Microsoft.Data.SqlClient.xml" />
       </group>
       <group targetFramework="netstandard2.1">
           <reference file="Microsoft.Data.SqlClient.dll" />
-          <reference file="Microsoft.Data.SqlClient.pdb" />
+          <!-- <reference file="Microsoft.Data.SqlClient.pdb" /> -->
```

Comment:
why pdbs are taken out of the nuget? This also raised a warning at build time: 

```
EXEC : warning : NU5131: References were found in the nuspec, but some reference assemblies were not found in both the
nuspec and ref folder. Add the following reference assemblies:

 - Add Microsoft.Data.SqlClient.pdb to the net462 reference group in the nuspec
  - Add Microsoft.Data.SqlClient.pdb to the net7.0 reference group in the nuspec
  - Add Microsoft.Data.SqlClient.pdb to the net6.0 reference group in the nuspec
  - Add Microsoft.Data.SqlClient.pdb to the netstandard2.0 reference group in the nuspec
  - Add Microsoft.Data.SqlClient.pdb to the netstandard2.1 reference group in the nuspec
```

Reply:
The pdb files were taken out of the reference section of MDS.nuspec file due to the error message during packaging and that packages were not generated.

_Attempting to build package from 'Microsoft.Data.SqlClient.nuspec'.
EXEC : error NU5018: Invalid assembly reference 'Microsoft.Data.SqlClient.pdb'. Ensure that a file named 'Microsoft.Data.SqlClient.pdb' exists in the lib directory. 
NuGet.Packaging.Core.PackagingException: Invalid assembly reference 'Microsoft.Data.SqlClient.pdb'. Ensure that a file named 'Microsoft.Data.SqlClient.pdb' exists in the lib directory._

However, without the pdbs in the reference section of the MDS.nuspec, a warning message below showed up after building the packages.

_EXEC : warning : NU5131: References were found in the nuspec, but some reference assemblies were not found in both the nuspec and ref folder. Add the following reference assemblies: 
  - Add Microsoft.Data.SqlClient.pdb to the net462 reference group in the nuspec
  - Add Microsoft.Data.SqlClient.pdb to the net7.0 reference group in the nuspec
  - Add Microsoft.Data.SqlClient.pdb to the net6.0 reference group in the nuspec
  - Add Microsoft.Data.SqlClient.pdb to the netstandard2.0 reference group in the nuspec
  - Add Microsoft.Data.SqlClient.pdb to the netstandard2.1 reference group in the nuspec_
  
Nevertheless, the package files, nupkg, and snupkg, were still generated with the expected contents and tested to be working,

Reply:
I suspect these errors are because the pdbs weren't removed from the files below (lines 142-275).